sklearn/compose/_column_transformer.py

```diff
@@ -467,11 +467,11 @@ def fit(self, X, y=None):
 
         Parameters
         ----------
-        X : array-like or DataFrame of shape [n_samples, n_features]
+        X : {array-like, dataframe} of shape (n_samples, n_features)
             Input data, of which specified subsets are used to fit the
             transformers.
 
-        y : array-like, shape (n_samples, ...), optional
+        y : array-like of shape (n_samples,), default=None
```

Comment:
`y` is not restricted to 1-D by the column transformer. They are passed along as is to the underlying transformers.

Reply:
Oh, you are right, corrected. Thank you.